src/main/java/net/fabricmc/api/EnvironmentInterface.java

```diff
@@ -0,0 +1,10 @@
+package net.fabricmc.api;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.CLASS)
+public @interface EnvironmentInterface {
```

Comment:
I just realized this annotation should probably be repeatable to allow specifying stripped interfaces for both SERVER and CLIENT.

Reply:
Yes